### PR TITLE
Audio tests

### PR DESCRIPTION
**Pull request by sam-itt**
Not for merging

### CODE REVIEW

File: samples/sdl_audio-s162441/main.c [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        return (1);
+    }
+
+
+    wave.spec.freq = 44100; 
+	wave.spec.format = AUDIO_S16; 
+	wave.spec.channels = 2; 
+	wave.spec.samples = 4096; 
+	wave.spec.callback = NULL;
+    wave.spec.userdata = NULL;
```
Comment (JayFoxRox) [inline samples/sdl_audio-s162441/main.c]: Tabs vs. spaces.

*Other instances, too.*

File: samples/sdl_audio-s16248/Makefile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+XBE_TITLE = sdl_audio_s16248
```
Comment (JayFoxRox) [inline samples/sdl_audio-s16248/Makefile]: These test names are really cryptic and not consistent.

File: samples/sdl_audio-48-441/Makefile [hunk 3]
```diff
@@ -0,0 +1,8 @@
+XBE_TITLE = sdl_audio_s162441
```
Comment (JayFoxRox) [inline samples/sdl_audio-48-441/Makefile]: Title does not match what the test does or path.

File: samples/sdl_audio-s162441/main.c [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+    debugPrint("Spec %p:\n", spec);
+    debugPrint("\tfreq: %d\n",spec->freq);
+    debugPrint("\tformat: %d\n",spec->format);
+    debugPrint("\tchannels: %d\n",spec->channels);
+    debugPrint("\tsilence: %d\n",spec->silence);
+    debugPrint("\tsamples: %d\n",spec->samples);
+    debugPrint("\tsize: %d\n",spec->size);
+    debugPrint("\tcallback: %p\n",spec->callback);
+    debugPrint("\tuserdata: %p\n",spec->userdata);
+}
```
Comment (JayFoxRox) [inline samples/sdl_audio-s162441/main.c]: This function is unused.

**Comment by sam-itt** [comment]
Not for merging, just for the actual tests.

**Pull request closed without merging**